config/jobconfig.toml

```diff
@@ -183,8 +183,8 @@
 "# Playtime Requirements" = 0
 "# Required Account Age" = 0
 "# Required Character Age" = 0
-"# Spawn Positions" = 3
-"# Total Positions" = 5
+"# Spawn Positions" = 4
+"# Total Positions" = 6
```

Comment:
The job datum, `medical_doctor.dm` too. I don't know if it matters but just in case.